api/src/main/webapp/WEB-INF/cron_perf.yaml

```diff
@@ -1,91 +1,5 @@
-  timezone: UTC
-  target: api
-# This cron job is disabled in perf env. See RW-6430 for details.
-#- description: 'Daily audit of gcp resources that users have access to'
-#  url: /v1/cron/bulkAuditProjectAccess
-#  schedule: every 1 hours
-#  timezone: UTC
-#  target: api
-- description: 'If the AoU Billing Project buffer is not full, refill with one or more projects.'
-  url: /v1/cron/bufferBillingProjects
-  schedule: every 1 minutes
-  timezone: UTC
-  target: api
-- description: 'Fetch a BillingProjectBufferEntry that is in the CREATING state and check its status on Firecloud'
-  url: /v1/cron/syncBillingProjectStatus
-  schedule: every 1 minutes
-  timezone: UTC
-  target: api
-- description: 'Find BillingProjectBufferEntries that have failed during the creation or assignment step and set their statuses to ERROR'
-  url: /v1/cron/cleanBillingBuffer
-  schedule: every 1 hours
-  timezone: UTC
-  target: api
-- description: 'Find and alert users that have exceeded their free tier billing usage'
-  url: /v1/cron/checkFreeTierBillingUsage
-  # This is a change specific to the perf environment. Rather than run every 30 mins, we reduce the frequency of the
-  # free-tier billing job to once per day at a time known to not conflict with perf tests. See RW-6430 for details.
-  schedule: every day 15:00
-  timezone: America/Chicago
-  target: api
-- description: >
-    Sample all gauge metrics for OpenCensus Monitoring and record them. The one-minute interval is
-    the highest granularity for Stackdriver Monitoring and the lowest interval for an AppEngine
-    cron job.
-  url: /v1/cron/monitoring/updateGaugeMetrics
-  schedule: every 1 minutes
-  timezone: UTC
-  target: api
-- description: >
-    Export user and workspace data to RDR.
-    RDR export is hard-coded to 9pm CT, to align with VUMC expectations that the daily export is run at a time
-    that is (1) after the close of normal business working hours, and (2) early enough in the evening that the
-    entire export (and downstream data flows) can complete before start of the next business day.
-  url: /v1/cron/exportToRdr
-  schedule: every day 22:00
-  timezone: America/Chicago
-  target: api
-- description: >
-    For each workspace update the attribute need_rp_prompt if it has been created 15 days/an Year earlier
-  url: /v1/cron/updateResearchPurposeReviewPrompt
-  schedule: every day 21:00
-  timezone: America/Chicago
-  target: api
-- description: >
-    Compile a snapshot of workbench data for reporting and upload it to  BigQuery.
-  url: /v1/cron/uploadReportingSnapshot
-  schedule: every day 19:00
-  timezone: America/Chicago
-  target: api
-- description: >
-    Check each user to ensure they still meet compliance guidelines. If they do not, remove them from access tiers.
-  url: /v1/cron/synchronizeUserAccess
-  schedule: every day 23:00
-  timezone: America/Chicago
-  target: api
+- url: /v1/cron/bulkAuditProjectAccess
+  aou_skip_reason: "RW-6430"
```

Comment:
nit: add JIRA title or a short description so we have at least a high level understanding of why these are removed w/o having to reference JIRA.

Reply:
I prefer this pattern I think. A comment here can go out of sync and it may need to be duplicated to multiple files. This might make sense in other cases, but for these the explanation for why these are disabled is too complicated to explain succinctly here.